ldes-server-domain/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/validation/DcatViewValidatorTest.java

```diff
@@ -0,0 +1,127 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.validation;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFParser;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class DcatViewValidatorTest {
+
+	private final static String validDcatView = """
+							@prefix dct:   <http://purl.org/dc/terms/> .
+							@prefix dcat:  <http://www.w3.org/ns/dcat#> .
+							@prefix foaf:  <http://xmlns.com/foaf/0.1/> .
+							@prefix org:   <http://www.w3.org/ns/org#> .
+							@prefix legal: <http://www.w3.org/ns/legal#> .
+							@prefix m8g:   <http://data.europa.eu/m8g/>
+							@prefix locn:  <http://www.w3.org/ns/locn#>
+							@prefix skos:  <http://www.w3.org/2004/02/skos/core#>
+
+							[] a dcat:DataService ;
+									dct:title "My geo-spatial view"@en ;
+									dct:description "Geospatial fragmentation for my LDES"@en ;
+									dct:license [
+											a dct:LicenseDocument ;
+											dct:type [
+													a skos:Concept;
+													skos:prefLabel "some public license"@en
+											]
+									] .
+			""";
+
+	private DcatViewValidator validator;
+
+	@BeforeEach
+	void setUp() {
+		validator = new DcatViewValidator();
+	}
+
+	@Test
+	void should_NotThrowAnything_when_Valid() {
+		final Model model = RDFParser.fromString(validDcatView).lang(Lang.TURTLE).build().toModel();
+		// noinspection DataFlowIssue
```

Comment:
This issue says very little. If this is going to remain to warns us, maybe we should have an external link poiting to the issue?